instrumentation/spring/spring-boot-actuator-autoconfigure-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/actuator/v2_0/OpenTelemetryMeterRegistryAutoConfiguration.java

```diff
@@ -32,4 +40,30 @@ public class OpenTelemetryMeterRegistryAutoConfiguration {
   public MeterRegistry otelMeterRegistry() {
     return MicrometerSingletons.meterRegistry();
   }
+
+  @Bean
+  static BeanPostProcessor postProcessCompositeMeterRegistry() {
```

Comment:
do we have a convention of making these methods `public`?

Reply:
the other bean post processor is also static: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/428aa281d532c7f1a1953c16c26974bc997acf88/instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/internal/instrumentation/jdbc/JdbcInstrumentationAutoConfiguration.java#L34